backend/open_webui/models/domains.py

```
@@ -0,0 +1,170 @@
+                return False
+
+    def get_available_domains(self) -> List[DomainModel]:
+        """Get list of all available domains from the domains table only"""
+        # Get domains from domains table only
+        return self.get_domains()
+
+    def get_available_domains_list(self) -> List[str]:
+        """Get list of all available domain names as strings from domains table only"""
+        return [domain.domain for domain in self.get_domains()]
+
+    def get_available_domains_list(self) -> list[str]:
```

Ruff (0.13.2)

Redefinition of unused `get_available_domains_list` from line 161

(F811)